cmd/http/main.go

```diff
@@ -6,32 +6,25 @@ package main
 import (
 	"context"
 	"fmt"
-	"io"
-	"io/ioutil"
 	"log"
-	"net/http"
 	"os"
 	"strconv"
 	"time"
 
-	"google.golang.org/grpc/credentials"
-
-	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
 	"github.com/mainflux/mainflux"
 	adapter "github.com/mainflux/mainflux/http"
 	"github.com/mainflux/mainflux/http/api"
+	initutil "github.com/mainflux/mainflux/internal/init"
+	"github.com/mainflux/mainflux/internal/init/mfserver"
+	"github.com/mainflux/mainflux/internal/init/mfserver/httpserver"
```

Comment:
This is quite redundant. I'd use `"github.com/mainflux/mainflux/internal/servers/http"`